seccomp/README.md

```diff
@@ -185,47 +348,37 @@ When writing a seccomp filter sometimes arguments are truncated by the operating
 
 https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt
 
-#### Seccomp escapes
+#### seccomp escapes
 
-* Syscall numbers are architecture dependant, so raw BPF filters are not very portable.
-Luckily docker abstract this issue away, so you don't need to worry about it if using docker seccomp profiles.
+Syscall numbers are architecture dependant. This limits the portability of BPF filters. Fortunately Docker profiles abstract this issue away, so you don't need to worry about it if using Docker seccomp profiles.
 
-* ptrace is disabled by default and you should not enable it because it allows bypassing seccomp.
-You can use this script to test for seccomp escapes through ptrace:
-https://gist.github.com/thejh/8346f47e359adecd1d53
+`ptrace` is disabled by default and you should avoid enabling it. This is because it allows bypassing of seccomp. You can use [this script](https://gist.github.com/thejh/8346f47e359adecd1d53) to test for seccomp escapes through `ptrace`.
 
-#### Differences between docker versions
+#### Differences between Docker versions
 
 * Seccomp is supported as of Docker 1.10.
 
-* Using `--privileged` disables seccomp in all versions of docker even if you specify a seccomp profile with it.
-In general you should try to avoid ever using `--privileged` because it does too many things.
-You can achieve the same goal with `--cap-add ALL --security-opt apparmor=unconfined --security-opt seccomp=unconfined`.
-If you need access to devices use `--device`.
+* Using the `--privileged` flag when creating a container disables seccomp in all versions of docker - even if you explicitly specify a seccomp profile. In general you should avoid using the `--privileged` flag as it does too many things. You can achieve the same goal with `--cap-add ALL --security-opt apparmor=unconfined --security-opt seccomp=unconfined`. If you need access to devices use `--device`.
```

Comment:
To differentiate from the next bullet, can we specifically reference this is with `docker run`